.github/workflows/ci.yml

```diff
@@ -2,9 +2,10 @@
 name: CI
 
 on:
+  workflow_dispatch:
   push:
-    branches: ["*"]
-    tags-ignore: ["v*"]  # Skip CI for releases
+    branches: ["**"]
```

Comment:
Oh, so things like `a/b` wouldn't match `*` but would match `**`?

Reply:
I have no idea why it works, to be 100% frank. But hey, YOLO?